sub-sites/wiki/content/developers/build.md

```diff
@@ -0,0 +1,28 @@
+# Setting up from source
+
+In order to set up or build the source code some of the projects from the yukino repository, additional beforehand setup is required.
+
+# Kazahana
+
+Requirements:
+- [Flutter SDK](https://docs.flutter.dev/get-started/install)
+- [Android SDK](https://developer.android.com/studio)
+- (Optional)[Github Desktop](https://desktop.github.com/)
+- (Optional)[Git](https://git-scm.com/downloads)
+
+To begin, you have three options for getting source:
+Option one:
+In your browser, navigate to the [Kazahana repo](https://github.com/yukino-org/kazahana) and press the green code button and download the zip.
+Option two(Github Desktop):
+In your browser, navigate to the [Kazahana repo](https://github.com/yukino-org/kazahana) and press the green code button and open with Github dekstop. 
+Option three(Git):
```

Comment:
Keep just the Git commands. Also use "Kazahana Repository" instead of "Kazahana repo".